Userland/Utilities/tail.cpp

```diff
@@ -49,12 +61,35 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     bool follow = false;
     size_t wanted_line_count = DEFAULT_LINE_COUNT;
+    bool start_from_end = true;
     StringView file;
 
     Core::ArgsParser args_parser;
     args_parser.set_general_help("Print the end ('tail') of a file.");
     args_parser.add_option(follow, "Output data as it is written to the file", "follow", 'f');
-    args_parser.add_option(wanted_line_count, "Fetch the specified number of lines", "lines", 'n', "number");
+    args_parser.add_option(Core::ArgsParser::Option {
+        .argument_mode = Core::ArgsParser::OptionArgumentMode::Required,
+        .help_string = "output the last NUM lines, instead of the last 10;"
+                       " or use -n +NUM to output starting with line NUM",
+        .long_name = "lines",
+        .short_name = 'n',
+        .value_name = "[+]NUM",
+        .accept_value = [&](StringView lines) -> ErrorOr<bool> {
+            Optional<int> value;
```

Comment:
The underlying type of this Optional should match the type of `wanted_line_count`. This also has the benefit of not allowing the user to enter negative numbers.